scripts/install

```diff
@@ -134,59 +125,61 @@ virtualenv --python=python3 --system-site-packages "${final_path}/venv"
 #=================================================
 # Install Kresus with npm
 #=================================================
-
 ynh_script_progression --message="Installing app"
+
 ynh_use_nodejs
 (
 	cd "$final_path"
 	chown -R "$app": "$final_path"
 
 	# ynh_npm only exists for YunoHost 3.8+
-	"$nodejs_path/npm" install --production --unsafe-perm
+	ynh_exec_warn_less "$nodejs_path/npm" install --production --unsafe-perm
 )
 
 #=================================================
 # Setup Kresus postgresql database
 #=================================================
+ynh_script_progression --message="Creating a PostgreSQL database..."
+
+db_name=$(ynh_sanitize_dbid --db_name=$app)
+db_user=$db_name
+db_pwd=$(ynh_app_setting_get --app=$app --key=psqlpwd)
```

Comment:
How does that work now that it is set before the db setup? And why do we need to set it again line 149?